sdk/src/main/scala/com/horizen/account/utils/AccountFeePaymentsUtils.scala

```diff
@@ -0,0 +1,59 @@
+package com.horizen.account.utils
+
+import com.horizen.account.proposition.AddressProposition
+import com.horizen.evm.{StateDB, TrieHasher}
+import com.horizen.utils.MerkleTree
+
+import java.math.BigInteger
+
+object AccountFeePaymentsUtils {
+  val DEFAULT_ACCOUNT_FEE_PAYMENTS_HASH: Array[Byte] = StateDB.EMPTY_ROOT_HASH
+
+  def calculateFeePaymentsHash(feePaymentsInfo: Seq[AccountBlockFeeInfo]): Array[Byte] = {
+    if(feePaymentsInfo.isEmpty) {
+      // No fees for the whole epoch, so no fee payments for the Forgers.
+      DEFAULT_ACCOUNT_FEE_PAYMENTS_HASH
+    } else {
+      // turn seq elements into leaves and compute merkel root hash
+      TrieHasher.Root(feePaymentsInfo.map(info => info.bytes).toArray)
+    }
+  }
+
+  def getForgersRewards(blockFeeInfoSeq : Seq[AccountBlockFeeInfo]): Seq[(AddressProposition, BigInteger)] = {
```

Comment:
let's add the batch of Unit tests similar to the `src/test/scala/com/horizen/SidechainStateTest.scala -> feePayments()`

Reply:
Ok added a new `AccountStateTest.scala` file